easybuild/easyconfigs/s/Sniffles/Sniffles-1.0.5-foss-2015b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Sniffles'
+version = '1.0.5'
+
+homepage = 'https://github.com/fritzsedlazeck/Sniffles'
+description = """Sniffles is a structural variation caller using third generation sequencing
+  (PacBio or Oxford Nanopore). It detects all types of SVs using evidence from split-read
+  alignments, high-mismatch regions, and coverage analysis."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/fritzsedlazeck/Sniffles/archive/']
+
+patches = ['Sniffles-1.0.5_CMakeLists.patch']
+
+builddependencies = [('CMake', '3.4.1')]
+
+skipsteps = ['install']
+
+separate_build_dir = True
+
+buildopts = ' && mkdir -p %(installdir)s && cp -r ./bin %(installdir)s/'
+
+sanity_check_paths = {
+    'files': ['bin/sniffles'],
+    'dirs': ['bin']
```

Comment:
since you're already checking for `bin/sniffles`, this is kind of useless

Also, the patch suggests there are libraries involved as well?
Why are you not installing those, any particular reason?